.deployment.config.json

```diff
@@ -31,11 +31,13 @@
     },
     "snyk_code": {
         "no_scan": false,
-        "configurations": {
-            "org": "coveo-admin-ui",
-            "path": "./",
-            "project_name": "plasma"
-        }
+        "configurations": [
+            {
+                "org": "coveo-admin-ui",
+                "path": "./",
+                "project_name": "plasma"
+            }
+        ]
```

Comment:
Should we have one configuration per package?

Reply:
:thinking: I'm not sure that's necessary